lib/v8debugapi.js

```diff
@@ -189,7 +190,7 @@ module.exports.create = function(logger_, config_, fileStats_) {
       var num = breakpoints[breakpoint.id].v8Breakpoint.number();
       var listener = onBreakpointHit.bind(
           null, breakpoint, function(err) {
-            delete listeners[num];
+            listeners[num].enabled = false;
```

Comment:
In general this is a good thing to reason about. In this instance, the listener is invoked synchronously by `handleDebugEvents` which includes a `try-catch` that will handle and log any errors that result from the listener not being present so I think we're ok.